.azure/pipelines/ci.yml

```
@@ -1,5 +1,10 @@
-# Don't run CI for this config yet. We're not ready to move official builds on to Azure Pipelines
-trigger: none
+# Configure which branches trigger builds
+trigger:
+  batch: true
+  branches:
+    include:
+    - release/*
```

consistency nit&hellip;
```suggestion
    - release/2.*
```